dex/testing/eth/harness.sh

```diff
@@ -111,7 +119,7 @@ cat > "${NODES_ROOT}/genesis.json" <<EOF
     "1D4F2ee206474B136Af4868B887C7b166693c194": {
         "balance": "11000000000000000000000"
     },
-    "1c8F908d99F62522F8E0AD9504FFcC187e5514ca": {
+    "946dfaB1AD7caCFeF77dE70ea68819a30acD4577": {
```

Comment:
Fixes funds for the `simnet-setup.sh` script's seed.

Reply:
What broke it?  A previous change of ours or geth 1.10.21?

Reply:
The eth wallet creation got the extra step where the seed is used to make a mnemonic first. So different address from the same seed.

Reply:
oh right, you or marton just fixed that in the nodeclient harness code too.